### DISCUSSION TITLE
Can I get refund for accidential purchace Github teams?

### DISCUSSION THREAD

**Discussion by zolanvari** [community/community#31535] [New to GitHub] [answered]
I accidentally bought a yearly Github team for two seats (~USD 88). I canceled as I noticed they deducted from my Paypal by downgrading to the free version. I haven't used it at all. Will GitHub refund me? Thanks

**Comment by Simran-B** [comment, marked as answer]
You should contact the GitHub support.
I think this is the right form:
https://support.github.com/contact/account

**Reply by zolanvari** [reply]
Thank you. I already did. I will update here as soon as I got any response from them.

**Reply by zhangzw218** [reply]
Did you get the refund successfully?

**Reply by fikriauliya** [reply]
How is their reply? @zolanvari

**Reply by zolanvari** [reply]
They have refunded later.
